app/services/plan_service/api/plans.rb

```diff
@@ -37,7 +37,7 @@ def create(community_id:, plan:)
     # All plans should come from the external plan service and that's
     # why this function is deprecated
     #
-    def create_initial_trial(community_id:, plan:)
+    def create_initial_trial(community_id:, plan: {})
```

Comment:
And int_api/marketplaces_controller also uses. This is deprecated because we should get all trials from charm, but currently we don't.